web/skins/classic/views/montage.php

```diff
@@ -190,10 +190,13 @@
   <div id="page">
     <div id="header">
 <?php
-    $html = '';
-    $flip = ( (!isset($_COOKIE['zmMonitorFilterBarFlip'])) or ($_COOKIE['zmMonitorFilterBarFlip'] == 'down')) ? 'up' : 'down';
-    $html .= '<a class="flip" href="#"><i id="mfbflip" class="material-icons md-18">keyboard_arrow_' .$flip. '</i></a>'.PHP_EOL;
-    $html .= '<div class="container-fluid" id="mfbpanel"'.( ( $flip == 'down' ) ? ' style="display:none;"' : '' ) .'>'.PHP_EOL;
+    $html = '<a class="flip" href="#" 
+             data-flip-сontrol-object="#mfbpanel" 
+             data-flip-сontrol-run-after-func="applyChosen" 
+             data-flip-сontrol-run-after-complet-func="changeScale">
```

Comment:
I'm not at all sure about the need to use changeScale() (in the code as it is now...) when collapsing/expanding a block. Because it just spoils it. For example, it resets "Layout" to "Freeform", etc.
Maybe now disable its use on the Montage page? And then change the function?

Reply:
Agreed, and in my branch here, I am working to fix that.  But we do need to rescale the streaming images due to the change in space available.